src/Sylius/Bundle/UserBundle/Console/Command/AbstractRoleCommand.php

```diff
@@ -16,15 +16,15 @@
 use Doctrine\Persistence\ObjectManager;
 use Sylius\Component\User\Model\UserInterface;
 use Sylius\Component\User\Repository\UserRepositoryInterface;
-use SyliusLabs\Polyfill\Symfony\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Helper\QuestionHelper;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
 use Symfony\Component\Console\Question\ChoiceQuestion;
 use Symfony\Component\Console\Question\Question;
 use Webmozart\Assert\Assert;
 
-abstract class AbstractRoleCommand extends ContainerAwareCommand
+abstract class AbstractRoleCommand extends Command
```

Comment:
This class still uses container, so it needs to refactored to use DI.

![image](https://github.com/Sylius/Sylius/assets/870747/f45a0f4d-76e4-4f55-8f65-69ac407e34d8)

Reply:
yes, I need to rework this part and use dependency injection... Tone of works for this PR.

Reply:
done